README.md

```diff
@@ -5,7 +5,7 @@ BE Mod 2 Week 4/5 Group Project
 
 "Monster Shop" is a fictitious e-commerce platform where users can register to place items into a shopping cart and 'check out'. Users who work for a merchant can mark their items as 'fulfilled'; the last merchant to mark items in an order as 'fulfilled' will be able to get "shipped" by an admin. Each user role will have access to some or all CRUD functionality for application models.
 
-Students will be put into 3 or 4 person groups to complete the project.\n
+Students will be put into 3 or 4 person groups to complete the project.
```

Comment:
Weird formatting here that I didn't do. Looks to be automatic. Feel free to not merge these